src/explanations/units_variables.yaml

```diff
@@ -0,0 +1,171 @@
+- title: Units
+  shortTitle: Units
+  key: "units"
+  details: |
+    - Ether Units
+      While using Ethereum, we may encounter different units, such as Wei and Gwei. In Solidity, we can use wei, gwei or ether to specify a subdenomination of Ether. 
+      Wei is the smallest unit of measurement. One Wei is worth 0.000000000000000001 Ether (or 10-18). Because it cannot deal with decimal numbers, the EVM (Ethereum Virtual Machine) employs Wei as its base unit. Gwei, another common unit, is 1000000000 Wei, or 0.000000001 (or 10-9) Ether. 
+      
+      - `1 Wei = 1;`
+      - `1 Kwei	=	1e3;`
+      - `1 Mwei	=	1e6;`
+      - `1 Gwei	=	1e9;`
+      - `1 Szabo	=	1e12;`
+      - `1 Finney	=	1e15;`
+      - `1 Ether	=	1e18;`
+      - `1 Kether	=	1e21;`
+      - `1 Mether	=	1e24;`
+      - `1 Gether	=	1e27;`
+      - `1 Tether	=	1e30;`
+      
+      Some of the above unit names honour figures like  Charles Babbage, Wei Dai, Ada Lovelace, Claude Shannon, Nick Szabo, and Hal Finney who made significant contributions to the development of the computer or the blockchain. The following code snippet demonstrates a simple function to convert the total `wei` and 'gwei' (default unit in Solidity) to Ether.
```

Comment:
```suggestion
      The following code snippet demonstrates a simple function to convert the total `wei` and 'gwei' (default unit in Solidity) to Ether.
```

Reply:
I got this information from internet,  hence added the same.